nuxeo-dm/nuxeo-platform-webapp-core/src/main/java/org/nuxeo/ecm/webapp/action/PopupHelper.java

```diff
@@ -340,16 +342,9 @@ private DocumentModel getFirstParentAfterDelete(DocumentModel doc) {
         return null;
     }
 
-    private boolean isDocumentDeleted(DocumentModel doc) {
-        // test if the document still exists in the repository
-        if (!documentManager.exists(doc.getRef())) {
-            return true;
-        }
-        // test if the document is in the trash
-        if (LifeCycleConstants.DELETED_STATE.equals(doc.getCurrentLifeCycleState())) {
-            return true;
-        }
-        return false;
+    protected boolean isPopupDocumentDescendantOfCurrentDocument() {
+        return currentPopupDocument != null && currentContainer != null
+                && currentPopupDocument.getPathAsString().startsWith(currentContainer.getPathAsString() + "/");
```

Comment:
In theory this is incorrect if `currentContainer` is the root, as it would give a path of `//`, but that's an edge case of no interest to us here.